docs/config-file.md

```diff
@@ -68,8 +74,17 @@ export default async function () {
   };
 }
 ```
+Inside the return statement, you can specify the following options:
+### Glee Core Configurations
 
-Every protocol has different configuration needs so each protocol has unique configurations:
+These configurations apply to Glee itself, rather than any specific protocol.
+
+|Field|Default|Description|
+|--|--|--|
+|glee.gleeDir|`.glee`|Sets the Glee directory. Your sources will be compiled here.|
+|glee.lifecycleDir|`lifecycle`|Path to the directory that stores your [lifecycle events](./lifecycle-events.md).|
+|glee.functionsDir|`functions`| Path to the directory that stores your [functions](./functions.md).|
+|glee.asyncapiFilePath|`asyncapi.(yaml \| yml \| json)`| Path to your AsyncAPI file. |
```

Comment:
These changes shouldn't be on your PR. Please merge the latest `master` into your branch (no rebase! 😄).